docs/api/README.md

```diff
@@ -21,10 +31,10 @@ Generic placeholders are defined as follows:
 
 ## Table of contents
 
-1. [Project](./project.md)
-2. [Dashboard](./dashboard.md)
-   1. [Specification](./dashboard.md#dashboard-specification)
-   2. [API definition](./dashboard.md#api-definition)
+1. [Project](../project)
+2. [Dashboard](../dashboard)
+   1. [Specification](../dashboard.md#dashboard-specification)
```

Comment:
Some links have `.md` some others doesn't have it, is it normal?

Reply:
Actually!
IMO this page didn't makes sense anymore.
But let's see what @eschabell thinks about it.